crates/oxc_linter/src/rules/eslint/no_continue.rs

```diff
@@ -0,0 +1,70 @@
+use crate::{context::LintContext, rule::Rule, AstNode};
+use oxc_ast::AstKind;
+use oxc_diagnostics::{
+    miette::{self, Diagnostic},
+    thiserror::Error,
+};
+use oxc_macros::declare_oxc_lint;
+use oxc_span::Span;
+
+#[derive(Debug, Error, Diagnostic)]
+#[error("eslint(no-continue): Unexpected use of `continue` statement.")]
+#[diagnostic(severity(warning), help("Do not use the `continue` statement."))]
+struct NoContinueDiagnostic(#[label] pub Span);
+
+#[derive(Debug, Default, Clone)]
+pub struct NoContinue;
+
+declare_oxc_lint!(
+    /// ### What it does
+    /// Disallow `continue` statements
+    ///
+    /// ### Why is this bad?
+    /// The continue statement terminates execution of the statements in the current iteration of the current or labeled loop, and continues execution of the loop with the next iteration. When used incorrectly it makes code less testable, less readable and less maintainable. Structured control flow statements such as if should be used instead.
+    ///
+    /// ### Example
+    /// ```javascript
+    /// var sum = 0,
+    //     i;
+    //
+    // for(i = 0; i < 10; i++) {
+    //     if(i >= 5) {
+    //         continue;
+    //     }
+    //
+    //     sum += i;
+    // }
+    /// ```
+    NoContinue,
+    correctness
```

Comment:
our categories are listed here, what about changing it to `style`?

https://github.com/oxc-project/oxc/blob/c72675e89eb544e164b008d11d6034a323b26fb7/crates/oxc_cli/src/command/lint.rs#L88-L96

Reply:
done